packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-verification-contents.ts

```diff
@@ -28,6 +28,10 @@ export const jurisdictionVerificationContents = (): TJurisdictionVerificationCon
                 'A recent utility bill (electricity, water or gas) or recent bank statement or government-issued letter with your name and address.'
             )}`,
         },
+        not_applicable: {
+            icon: 'IcNotApplicableVerification',
+            text: 'Not Applicable',
```

Comment:
where do we use this **"text: 'Not Applicable"**?

Reply:
this text is not used, and not required. i have replaced it with an empty string.